src/shogun/mathematics/linalg/LinalgBackendEigen.h

```diff
@@ -95,6 +95,15 @@ class LinalgBackendEigen : public LinalgBackendBase
 	DEFINE_FOR_ALL_PTYPE(BACKEND_GENERIC_IN_PLACE_ADD, SGMatrix)
 	#undef BACKEND_GENERIC_IN_PLACE_ADD
 
+	/** Implementation of @see LinalgBackendBase::apply */
+	#define BACKEND_GENERIC_IN_PLACE_APPLY(Type, Container) \
+	virtual void apply(SGMatrix<Type>& a, Container<Type>& b, Container<Type>& result) const \
```

Comment:
That's true, thanks for pointing it out.

Maybe then the name dot as in numpy? I don't like apply too much...